packages/builder/package.json

```diff
@@ -26,7 +26,7 @@
     "lint:fix": "lint --fix",
     "lint": "lint",
     "test": "mocha",
-    "coverage": "nyc.js ../../node_modules/mocha/bin/mocha",
+    "coverage": "../../node_modules/nyc/bin/nyc.js ../../node_modules/mocha/bin/mocha",
```

Comment:
remove the `.js` from `nyc.js` then it will work. The original command didn't work, because there is no `nyc.js` in bin. The correct form should be: 

```
"coverage": "nyc ../../node_modules/mocha/bin/mocha",
```